src/main/java/com/server/SimpleGet.java

```diff
@@ -27,7 +27,7 @@ public void runServer() throws IOException {
 
             PrintWriter out = openOutputStream(clientSocket);
 
-            if(requestRouter.checkPath(requestHeaderParser.getPath())){
+            if(requestRouter.getResponseCode(requestHeaderParser.getPath(), requestHeaderParser.getMethod()) == 200){
```

Comment:
We can see that passing in the entire `RequestHeaderParser` object would reduce the data clump code smell, however, we don't want to create a dependency on the object, which is more likely to change in the future.